hack/tests/e2e-ansible-molecule.sh

```diff
@@ -43,7 +43,7 @@ cat "$ROOTDIR/test/ansible-memcached/watches-finalizer.yaml" >> watches.yaml
 header_text "Append v1 kind to watches to test watching already registered GVK"
 cat "$ROOTDIR/test/ansible-memcached/watches-v1-kind.yaml" >> watches.yaml
 echo $marker >> watches.yaml
-sed -i'.bak' -e '/- secrets/a \ \ - services' config/rbac/role.yaml; rm -f config/rbac/role.yaml.bak
+sed -i'.bak' -e '/- secrets/a \ \ \ \ \ \ - services' config/rbac/role.yaml; rm -f config/rbac/role.yaml.bak
```

Comment:
What crazy incantation is this? :) looks fine

Reply:
yamllint didnt like the lack of indentation, so we indented the rest of the role.yaml, which means this had to be pushed over too. :fire: